### ISSUE TITLE
Add sync varient of the api

### ISSUE THREAD

**Issue by sigmaSd** [c4spar/deno-dzx#27]
Currently the api is verbose to use because you have to await each call, this is especially noticeable if you use dzx repl.

The latest deno version introduced a synchronous process spawning api which can be used to add this feature https://deno.com/blog/v1.21#synchronous-subprocess-execution

And in my opinion making it the default interface is ver y reasonable

**Comment by c4spar** [comment]
Yep makes sence. I was waiting for the new api to replace the old with the new one.
